docs/guides/performance/README.md

```diff
@@ -72,4 +72,7 @@ Note: `stableId` feature is only available in versions above `1.4.0`
 Please note that lower value is better. A lower values ensures that less number of views are created and that they're quickly available for recycling. You should be choosing the smallest value that gives you zero blank spaces while scrolling.
 Larger values mount extra views and increases the offset after which views are available for recycling but it might help with faster scrolling depending on the use case.
 
-In most cases, defaults work the best.
\ No newline at end of file
+In most cases, defaults work the best.
+
+### 7) `key` prop is not needed!
+Please **don't** add `key` prop to the output of `rowRenderer`. It is intentional to not have it. Adding it will stop recycling and cause random mounts/unmounts. It will totally ruin performance.
```

Comment:
@naqvitalha 
Damn.. I've always had a key to the output of my rowRenderer. Has it always been like this or is it something being introduced with stable Ids?

Reply:
It has always been that way :) Get rid of it, You will see a HUGE perf boost.

Reply:
I moved it to point 2 based on your experience :)